torch/quantization/fx/qconfig_utils.py

```diff
@@ -97,3 +97,89 @@ def get_qconfig(qconfig_dict, module_type, module_name, global_qconfig):
     module_name_qconfig = get_module_name_qconfig(
         qconfig_dict, module_name, module_name_regex_qconfig)
     return module_name_qconfig
+
+def check_is_valid_qconfig_dict(qconfig_dict: Any) -> None:
+    r""" Checks if the given qconfig_dict has the correct keys
+
+    Args:
+      `qconfig_dict`: dictionary whose keys we want to check
+    """
+
+    qconfig_dict_allowed_keys = {"", "object_type", "module_name_regex", "module_name"}
```

Comment:
can we make this an argument of the function and make this a more general function instead of having 4 functions?

for example: we can have a `check_is_valid_config_dict(config_dict: Dict[str, Any], allowed_keys: Set[str])`

Reply:
Ooh that's a good idea -- I will do so right now!

Reply:
Do you think it would be better to still keep those 4 functions to pass in the specific allowed keys? This way, the specific allowed keys could be hidden from people looking into the quantize.py file.

Another idea could be to have the allowed keys as global variables in quantize.py which can then be referenced in the test cases. Right now I have just declared the list of allowed keys again in the test files. What are your thoughts on this?

Reply:
yeah, keeping the functions sounds OK I think

Reply:
I see, I think we can keep those 4 functions, also we can remove references to the list of allowed keys in test cases as well so that when we add new keys we don't need to update the test case.
I feel it's better to avoid having global variables in this case.